views/templates/admin/controllers/module_catalog/includes/tab-module-line.html.twig

```diff
@@ -39,7 +39,7 @@
           - <span class="module-badge-bought help-tooltip text-warning" data-title="{{ 'You bought this module on PrestaShop Addons. Thank You.'|trans({}, 'Modules.Mbo.Modulescatalog') }}"><i class="icon-pushpin"></i> <small>{{ 'Bought'|trans({}, 'Modules.Mbo.Modulescatalog') }}</small></span>
         {% elseif module.attributes.type is not empty and module.attributes.type == 'addonsMustHave' %}
           - <span class="module-badge-popular help-tooltip text-primary" data-title="{{ 'This module is available on PrestaShop Addons.'|trans({}, 'Modules.Mbo.Modulescatalog') }}"><i class="icon-group"></i> <small>{{ 'Popular'|trans({}, 'Modules.Mbo.Modulescatalog') }}</small></span>
-        {% elseif module.attributes.type is not empty and module.attributes.type == 'addonsPartner' %}
+        {% elseif module.attributes.author is not empty and module.attributes.author == 'PrestaShop Partners' %}
```

Comment:
👌🏾 

If one day we have to use `PrestaShop Partners` again, we will put it in a constant